<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="MainMenu">
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>1011</width>
          <height>661</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1011</width>
           <height>661</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
   font-weight: 700;
  color: white;


  text-decoration: none;
  padding: .8em 1em calc(.8em + 3px);
padding-bottom:5%;
  border-radius: 3px;
  background: rgb(64,199,129);
  box-shadow: 0 -3px rgb(53,167,110) inset;

        margin-top: -5%;

}</string>
            </property>
            <property name="text">
             <string>Удалить работника</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
   font-weight: 700;
  color: white;


  text-decoration: none;
  padding: .8em 1em calc(.8em + 3px);
padding-bottom:5%;
  border-radius: 3px;
  background: rgb(64,199,129);
  box-shadow: 0 -3px rgb(53,167,110) inset;

        margin-top: -5%;

}</string>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
   font-weight: 700;
  color: white;


  text-decoration: none;
  padding: .8em 1em calc(.8em + 3px);
padding-bottom:5%;
  border-radius: 3px;
  background: rgb(64,199,129);
  box-shadow: 0 -3px rgb(53,167,110) inset;

        margin-top: -5%;

}</string>
            </property>
            <property name="text">
             <string>Средняя зарплата</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
   font-weight: 700;
  color: white;


  text-decoration: none;
  padding: .8em 1em calc(.8em + 3px);
padding-bottom:5%;
  border-radius: 3px;
  background: rgb(64,199,129);
  box-shadow: 0 -3px rgb(53,167,110) inset;

        margin-top: -5%;

}</string>
            </property>
            <property name="text">
             <string>Показать по условию</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
   font-weight: 700;
  color: white;


  text-decoration: none;
  padding: .8em 1em calc(.8em + 3px);
padding-bottom:5%;
  border-radius: 3px;
  background: rgb(64,199,129);
  box-shadow: 0 -3px rgb(53,167,110) inset;

        margin-top: -5%;

}</string>
            </property>
            <property name="text">
             <string>Добавить работника</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
             <property name="font">
              <font>
               <strikeout>false</strikeout>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="addworker">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2">
         <widget class="QLineEdit" name="name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Отдел</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Зарплата</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="surname"/>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="department"/>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="fathername"/>
        </item>
        <item row="7" column="2">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="salary"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Имя</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Введите данные работника</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="delete_worker">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="toolTip">
           <string comment="чччч"/>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Введите номер работника, которого вы хотите удалить из базы.</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>989</width>
              <height>611</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
             <property name="font">
              <font>
               <strikeout>false</strikeout>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Вернуться в меню</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="show_if">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="2">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Зарплата меньше, либо равна введённой</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="placeholderText">
           <string>Введите величину зарплаты, чтобы показать работников...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <widget class="QTableWidget" name="tableWidget_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>989</width>
              <height>621</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Зарплата больше, либо равна введённой</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Вернуться в меню</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="find">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="placeholderText">
           <string>Выберите параметр для поиска в выпадающем меню справа и введите в данное поле нужное Вам значение.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QScrollArea" name="scrollArea_4">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <widget class="QTableWidget" name="tableWidget_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>989</width>
              <height>621</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="3">
         <widget class="QPushButton" name="pushButton_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Выйти в меню</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Имя</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Отчество</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Отдел</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="median"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
